src/content/enterprise/install/releases.mdx

```diff
@@ -5,6 +5,36 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.0.0
+September 1, 2022
+
+### Removal Notice
+- Kubernetes [1.19](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md) is no longer supprted.
+
+### Deprecation Notice
+- Support for Kubernetes [1.20](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.20.md) has been derprecated and will be removed next release.
+
+### Features
+- License enforcements changed to allow for easier adoption.
+    - Unlicensed instances will be limited to 1 user.
+    - Instances with a trial license will be limted to 3 users.
```

Comment:
This is not the case right? The limit should come from the license? (Eg we could do a trial with 100 seats for a large team)

Reply:
This is an incorrect statement. The line should be removed and trial is simply a boolean in the license data and has no bearing on the seat limits.